### PR TITLE
PCF-222 Results: Add replacement for search after title/test name

### PR DESCRIPTION
**Pull request by esanuandra**
This PR adds the components for Search by title and Apply button in disabled mode. 

To see test you have to go on http://localhost:3000/#/beta/compare-results

### CODE REVIEW

**Review by beatrice-acasandrei — APPROVED** [review]

File: src/__tests__/CompareResults/beta/CompareResultsView.test.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import CompareResultsViewBeta from '../../../components/CompareResults/beta/CompareResultsView';
+import ResultsViewBeta from '../../../components/CompareResults/beta/ResultsView';
```
Comment (beatrice-acasandrei) [inline src/__tests__/CompareResults/beta/CompareResultsView.test.tsx:1]: For consistency we should use `ResultsViewBeta` or `CompareResultsViewBeta` when importing. I'm ok with either version.

```suggestion
import CompareResultsViewBeta from '../../../components/CompareResults/beta/ResultsView';
```

File: src/components/CompareResults/beta/SearchInput.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    $nest: {
+      '.form-container': {
+        maxWidth: '447px',
+      },
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      '.MuiButtonBase-root': {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        ...ButtonsLightRaw.Secondary,
+        marginLeft: Spacing.Small,
+      },
```
Comment (esanuandra) [inline src/components/CompareResults/beta/SearchInput.tsx:20]: @Carla-Moz do you have an idea how to avoid these eslint comments in future PRs?

Comment (Carla-Moz) [reply src/components/CompareResults/beta/SearchInput.tsx:20]: Ah, yes, I'll create an issue for that. We probably have to update something in ts config files for these style files. I'll look into it. Maybe exclude the styles directory.

**Comment by esanuandra** [comment]
image

File: src/components/CompareResults/beta/SearchInput.tsx (hunk 2 above)
Comment (Carla-Moz) [reply src/components/CompareResults/beta/SearchInput.tsx:20]: @esanuandra can you try updating your npm to latest version and see if that works. What version are you using now?

**Pull request merged**